psalm.xml

```diff
@@ -38,6 +38,8 @@
             </errorLevel>
         </UndefinedConstant>
         <RedundantPropertyInitializationCheck errorLevel="suppress"/>
+        <RedundantCast errorLevel="suppress"/>
         <UndefinedAttributeClass errorLevel="suppress"/>
+        <TooManyTemplateParams errorLevel="suppress"/>
```

Comment:
Suppressing this can cause actual errors to be hidden. If there is a mismatch in template params for a particular interface/class between psalm and phpstan then only that particular error should be added to the baseline.